### PR TITLE
MGDCTRS-616, MGDCTRS-567:feat:address UX recommendation

### PR DESCRIPTION
**Pull request by uidoyen**
https://issues.redhat.com/browse/MGDCTRS-616
As per discussion in the UX call, this feature is implemented just for duplicate connectors.

Screenshot from 2022-04-22 14-43-13

https://issues.redhat.com/browse/MGDCTRS-567
Synchronizes the title with sub-step label

### CODE REVIEW

**Comment by indraraj** [comment]
Hey @uidoyen I think we decided not to use the "category" word. so it will be like "**Connector**: Elasticsearch sink"  
@jgiardino can confirm.

File: src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx [hunk 1]
```diff
@@ -101,9 +105,15 @@ export const DuplicateConnectorPage: FunctionComponent<DuplicateConnectorPagePro
             </BreadcrumbItem>
             <BreadcrumbItem isActive>{t('duplicateConnector')}</BreadcrumbItem>
           </Breadcrumb>
-          <Level className={'pf-u-pt-md pf-u-pb-md'}>
+          <TextContent className={'pf-u-pt-md pf-u-pb-md'}>
             <Title headingLevel="h1">{t('duplicateConnector')}</Title>
-          </Level>
+            {connectorData && connectorTypeDetails ? (
+              <Text>
+                <strong>Connector Category: </strong>
```
Comment (indraraj) [inline src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx:112]: we could use i18 txt for this

**Comment by indraraj** [comment]
for DBZ connector type is not showing 
image

File: src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx (hunk 1 above)
Comment (uidoyen) [reply src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx]: Nice catch, Thanks!

File: src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx [hunk 2]
```diff
@@ -29,7 +31,9 @@ import {
   useBasename,
   useConfig,
 } from '@rhoas/app-services-ui-shared';
-import { Connector, ConnectorType } from '@rhoas/connector-management-sdk';
+import { Connector, ConnectorTypeAllOf } from '@rhoas/connector-management-sdk';
+
+// import { useConnectorsMachine } from '../ConnectorsPage/ConnectorsPageContext';
```
Comment (indraraj) [inline src/app/pages/CreateConnectorPage/DuplicateConnectorPage.tsx:36]: could remove this

**Review by indraraj — APPROVED** [review]
Tested locally, looks great.

**Pull request merged**